src/tests/ftest/launch.py

```diff
@@ -18,34 +15,44 @@
 import site
 import sys
 import time
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from collections import OrderedDict, defaultdict
+from tempfile import TemporaryDirectory
 
 # When SRE-439 is fixed we should be able to include these import statements here
 # from avocado.core.settings import settings
 # from avocado.core.version import MAJOR, MINOR
 # from avocado.utils.stacktrace import prepare_exc_info
 from ClusterShell.NodeSet import NodeSet
-
 # When SRE-439 is fixed we should be able to include these import statements here
 # from util.distro_utils import detect
 # pylint: disable=import-error,no-name-in-module
-from process_core_files import CoreFileProcessing, CoreFileException
+from process_core_files import CoreFileException, CoreFileProcessing
 from slurm_setup import SlurmSetup, SlurmSetupException
 
 # Update the path to support utils files that import other utils files
+# This is not good coding practice. Should use package paths and remove alls these E402.
```

Comment:
> GitHubActions / Codespell
>
> alls ==> all, falls

This line introduced a new codespell error and should have been fixed before merging

Reply:
oops sorry for the typo.

Reply:
Just to expand on this, many of our CI checks check the tree, not the diff so this warning will be reported for all PRs until fixed.